src/routes/(user)/clients/addClient/+page.server.ts

```diff
@@ -0,0 +1,37 @@
+import { superValidate } from 'sveltekit-superforms';
+import { zod } from 'sveltekit-superforms/adapters';
+import { z } from 'zod';
+import { KVConfig } from '$lib/server/config/kvConfig';
+import { getSystemDatabaseStrategy } from '$lib/server/database';
+import { ClientRepository } from '$lib/server/repository/clients';
+
+const schema = z.object({
+    name: z.string()
+});
+
+export const load = (async () => {
+    const form = await superValidate(zod(schema));
+    return { form };
+});
+
+import { message } from 'sveltekit-superforms';
+import { fail } from '@sveltejs/kit';
+
+export const actions = {
+  default: async ({ request } : {request : Request}) => {
```

Comment:
The type definition should not be there. You can change the type of the `actions` variable to the generated type. See other files for reference / SvelteKit docs for reference.